unpacker.json

```diff
@@ -4357,48 +4357,6 @@
 			"expectedmax": 0.0
 		}
 	},
-	"0x35C": {
-		"ECU_System_Status": {
-			"type": "Uint",
-			"startbyte": 0,
-			"length": 2,
-			"factor": 1.0,
-			"displayname": "",
-			"unit": "",
-			"expectedmin": 0.0,
-			"expectedmax": 0.0
-		},
-		"ECU_Torque_Allocation": {
-			"type": "Uint",
-			"startbyte": 2,
-			"length": 1,
-			"factor": 1.0,
-			"displayname": "",
-			"unit": "",
-			"expectedmin": 0.0,
-			"expectedmax": 0.0
-		},
```

Comment:
Wouldnt removing these destroy old Atmos / ELD logs? As Im aware, you also need the correct unpacker for the logs stored from previous years?

Reply:
As of now, yes. So we will wait a bit with this I think.